include/myst/futex.h

```diff
@@ -18,12 +18,21 @@
 #define FUTEX_UNLOCK_PI      7
 #define FUTEX_TRYLOCK_PI     8
 #define FUTEX_WAIT_BITSET    9
+#define FUTEX_WAKE_BITSET    10
 #define FUTEX_PRIVATE        128
 #define FUTEX_CLOCK_REALTIME 256
 // clang-format on
 
+#define FUTEX_BITSET_MATCH_ANY 0xffffffff
+
 int myst_futex_wait(int* uaddr, int val, const struct timespec* to);
+int myst_futex_wait_ops(
```

Comment:
Please add blank line before this. I wish the formatter could do that but unfortunately it cannot.